src/cli/README.md

```diff
@@ -114,7 +114,7 @@ Done
 
 ### extpanid \<extpanid\>
 
-Set the Thread Extended PAN ID value.
```

Comment:
I agree.  All of these configuration parameters are defined by the Thread standard and are not specific to OpenThread.